src/VisualStudio/Core/Next/FindReferences/StreamingFindReferencesPresenter.TableDataSourceFindReferencesContext.cs

```diff
@@ -355,13 +355,13 @@ private TextSpan GetRegionSpanForReference(SourceText sourceText, TextSpan refer
                     sourceText.Lines[lastLineNumber].End);
             }
 
-            private async Task<TaggedTextAndHighlightSpan> GetTaggedTextForReferenceAsync(
+            private async Task<ClassifiedSpansAndHighlightSpan> GetTaggedTextForReferenceAsync(
                 Document document, TextSpan referenceSpan, TextSpan widenedSpan, CancellationToken cancellationToken)
             {
                 var classificationService = document.GetLanguageService<IEditorClassificationService>();
                 if (classificationService == null)
                 {
-                    return new TaggedTextAndHighlightSpan(ImmutableArray<TaggedText>.Empty, new TextSpan());
+                    return new ClassifiedSpansAndHighlightSpan(ImmutableArray<ClassifiedSpan>.Empty, new TextSpan());
                 }
```

Comment:
Does it make sense to have a dedicated `ClassifiedSpansAndHighlightSpan.Default` rather than calling `new`?